### PR TITLE
chore(css): Move CSS examples - layout cookbooks

### PR DESCRIPTION
**Pull request by OnkarRuikar**
- part of https://github.com/mdn/mdn/issues/597

This PR moves some CSS external examples into content as part of an initiative to reduce repo maintenance and code duplication.

### CODE REVIEW

File: files/en-us/web/css/layout_cookbook/breadcrumb_navigation/index.md [hunk 1]
```diff
@@ -16,7 +16,40 @@ Display the hierarchy of the site by displaying inline links, with a separator b
...
+    <li><a href="#">Home</a></li>
+    <li><a href="#">Category</a></li>
+    <li><a href="#">Sub Category</a></li>
+    <li><a href="#">Type</a></li>
+    <li><span aria-current="page">Product</span></li>
+  </ul>
+</nav>
+```
+
+```css live-sample___breadcrumb-example
```
Comment (bsmth) [inline files/en-us/web/css/layout_cookbook/breadcrumb_navigation/index.md:31]: Not sure what font the screenshot is using, but we should use a sans-serif at least. I've been using the following in other examples:

```suggestion
```css live-sample___breadcrumb-example
body {
  font: 1.2em sans-serif;
}
```

File: files/en-us/web/css/layout_cookbook/list_group_with_badges/index.md [hunk 2]
```diff
@@ -16,7 +16,63 @@ The list items should be displayed with the badges. The badge should be aligned
...
+  </li>
+  <li>
+    Item Four
+    <span class="badge">0</span>
+  </li>
+</ul>
+```
+
+```css live-sample___list-group-badges-example
+.list-group {
```
Comment (bsmth) [inline files/en-us/web/css/layout_cookbook/list_group_with_badges/index.md:41]: We can add the `body` rules to fix some oddities: 

https://github.com/mdn/css-examples/blob/82dc5e69fa9b5db32c794dbeb794df5a0e0ac007/css-cookbook/list-group-badges--download.html#L9C7-L19C8

I would add this to the others, in general, to keep the font looking like the screenshots: 

```css
body {
  font: 1.2em sans-serif;
  /*   or  */
  font: 1.2em / 1.5 sans-serif;
}
```

File: files/en-us/web/css/layout_cookbook/breadcrumb_navigation/index.md [hunk 3]
```diff
@@ -16,7 +16,40 @@ Display the hierarchy of the site by displaying inline links, with a separator b
 
 ## Recipe
 
-{{EmbedGHLiveSample("css-examples/css-cookbook/breadcrumb-navigation.html", '100%', 530)}}
+```html live-sample___breadcrumb-example
```
[the remark this answers is not in the document]
Comment (OnkarRuikar) [reply files/en-us/web/css/layout_cookbook/breadcrumb_navigation/index.md:21]: done

File: files/en-us/web/css/layout_cookbook/breadcrumb_navigation/index.md (hunk 1 above)
Comment (OnkarRuikar) [reply files/en-us/web/css/layout_cookbook/breadcrumb_navigation/index.md:33]: done

File: files/en-us/web/css/layout_cookbook/list_group_with_badges/index.md (hunk 2 above)
Comment (OnkarRuikar) [reply files/en-us/web/css/layout_cookbook/list_group_with_badges/index.md:43]: Done. Also, the screenshots look a bit like a zoomed-in version of the output so we can't do much to match the new code.

File: files/en-us/web/css/layout_cookbook/breadcrumb_navigation/index.md [hunk 4]
```diff
@@ -16,11 +16,45 @@ Display the hierarchy of the site by displaying inline links, with a separator b
...
+.breadcrumb {
+  padding: 0 0.5rem;
+}
+
+.breadcrumb ul {
+  display: flex;
+  flex-wrap: wrap;
+  list-style: none;
+  margin: 0;
+  padding: 0;
```
Comment (bsmth) [inline files/en-us/web/css/layout_cookbook/breadcrumb_navigation/index.md:47]: This looks a bit odd, what do you think about this?

```suggestion
  padding: 0;
  align-items: end;
```

File: files/en-us/web/css/layout_cookbook/card/index.md [hunk 5]
```diff
@@ -20,11 +20,123 @@ The cards in the group should line up in two dimensions — both vertically and
...
+body {
+  font: 1.2em sans-serif;
+}
+
+img {
+  max-width: 100%;
+}
+
+.cards {
+  max-width: 800px;
```
Comment (bsmth) [inline files/en-us/web/css/layout_cookbook/card/index.md:101]: We can get rid of scrollbars:

```suggestion
  max-width: 700px;
```

File: files/en-us/web/css/layout_cookbook/grid_wrapper/index.md [hunk 6]
```diff
@@ -14,11 +14,66 @@ Items placed on the grid should be able to align to a horizontally-centered max-
...
+    </p>
+  </div>
+  <div class="right-wrapper">
+    <p>This item aligns to the right edge of the “wrapper” columns.</p>
+  </div>
+</div>
+```
+
+```css live-sample___grid-wrapper-example
+.grid {
```
Comment (bsmth) [inline files/en-us/web/css/layout_cookbook/grid_wrapper/index.md:43]:
```suggestion
body {
  font: 1.2em sans-serif;
}
.grid {
```

File: files/en-us/web/css/layout_cookbook/list_group_with_badges/index.md (hunk 2 above)
Comment (bsmth) [reply files/en-us/web/css/layout_cookbook/list_group_with_badges/index.md:43]: Still looks a little weird, how's this:

```css
body {
  font: 1.2em / 1.5 sans-serif;
  padding: 0;
  margin: 1em;
}
.list-group {
  list-style: none;
  margin: 0;
  padding: 0;
  border: 1px solid #ccc;
  border-radius: 0.5em;
  width: 20em;
}

.list-group li {
  border-top: 1px solid #ccc;
  padding: 0.5em;
  display: flex;
  justify-content: space-between;
  align-items: center;
}

.list-group li:first-child {
  border-top: 0;
}

.list-group .badge {
  background-color: rebeccapurple;
  color: #fff;
  font-weight: bold;
  font-size: 80%;
  border-radius: 10em;
  min-width: 1.5em;
  padding: 0.25em;
  text-align: center;
}
```

File: files/en-us/web/css/layout_cookbook/list_group_with_badges/index.md [hunk 7]
```diff
@@ -16,11 +16,66 @@ The list items should be displayed with the badges. The badge should be aligned
...
+.list-group .badge {
+  background-color: rebeccapurple;
+  color: #fff;
+  font-weight: bold;
+  font-size: 80%;
+  border-radius: 10em;
+  min-width: 1.5em;
+  padding: 0.25em;
+  text-align: center;
+}
```
Comment (bsmth) [inline files/en-us/web/css/layout_cookbook/list_group_with_badges/index.md:75]:
```suggestion
body {
  font: 1.2em / 1.5 sans-serif;
  padding: 0;
  margin: 1em;
}
.list-group {
  list-style: none;
  margin: 0;
  padding: 0;
  border: 1px solid #ccc;
  border-radius: 0.5em;
  width: 20em;
}

.list-group li {
  border-top: 1px solid #ccc;
  padding: 0.5em;
  display: flex;
  justify-content: space-between;
  align-items: center;
}

.list-group li:first-child {
  border-top: 0;
}

.list-group .badge {
  background-color: rebeccapurple;
  color: #fff;
  font-weight: bold;
  font-size: 80%;
  border-radius: 10em;
  min-width: 1.5em;
  padding: 0.25em;
  text-align: center;
}
```

File: files/en-us/web/css/layout_cookbook/media_objects/index.md [hunk 8]
```diff
@@ -22,11 +22,151 @@ Media Object pattern needs some or all of the following characteristics:
...
+      vitae ligula sit amet maximus. Nunc auctor neque ipsum, ac porttitor elit
+      lobortis ac. Vivamus ultrices sodales tellus et aliquam. Pellentesque
+      porta sit amet nulla vitae luctus. Praesent quis risus id dolor venenatis
+      condimentum.
+    </p>
+  </div>
+  <div class="footer"></div>
+</div>
+
+<div class="media media-flip">
```
Comment (bsmth) [inline files/en-us/web/css/layout_cookbook/media_objects/index.md:66]: HTML is a little long, so we could break it up / hide some of it:

````suggestion
</div>
```

```html hidden live-sample___media-objects-example
<div class="media media-flip">
````

File: files/en-us/web/css/layout_cookbook/pagination/index.md [hunk 9]
```diff
@@ -18,11 +18,90 @@ Typically, the pagination component will be centered horizontally underneath the
...
+.pagination a[aria-current="page"] {
+  background-color: #333;
+  color: #fff;
+}
+```
 
-> [!CALLOUT]
->
-> [Download this example](https://github.com/mdn/css-examples/blob/main/css-cookbook/pagination--download.html)
+{{EmbedLiveSample("pagination-example", "", "200px")}}
```
Comment (bsmth) [inline files/en-us/web/css/layout_cookbook/pagination/index.md:104]: I think we can do this one:

```suggestion
{{EmbedLiveSample("pagination-example")}}
```

File: files/en-us/web/css/layout_cookbook/split_navigation/index.md [hunk 10]
```diff
@@ -16,11 +16,40 @@ A common navigation pattern is to have one element pushed away from the others.
...
+  padding: 0.5em 1em;
+  display: block;
+}
+
+.push {
+  margin-left: auto;
+}
+```
+
+{{EmbedLiveSample("split-navigation-example", "", "200px")}}
```
Comment (bsmth) [inline files/en-us/web/css/layout_cookbook/split_navigation/index.md:52]: Same here:

```suggestion
{{EmbedLiveSample("split-navigation-example")}}
```

**Comment by bsmth** [comment]
Looking good, couple of comments for you to check out, then we should be ready to go! 👍🏻

**Review by bsmth — APPROVED** [review]
LGTM, thanks, Onkar!

**Pull request merged**